### DISCUSSION TITLE
Timebase plugin for Grafana

### DISCUSSION THREAD

**Discussion by sujit-sarjerao-kamthe-db** [finos/TimeBase-CE#276] [Q&A]
Hello Team,

I am trying to add Timebase plugin into the Grafana, but getting below error,  and due to that plugin is not getting displayed on Grafana GUI page 

logger=plugin.loader t=2024-04-20T12:57:52.147070556+02:00 level=info msg="Plugin registered" pluginID=input
logger=plugin.loader t=2024-04-20T12:57:52.147537705+02:00 level=warn msg="Skipping loading plugin due to problem with signature" pluginID=epam-timebase-datasource status=unsigned

Timebase version :   QuantServer Version:  6.1.23
Grafana version  :   grafana-9.3.0
Timebase plugin  :  1.0.12

Please suggest, how i can use unsigned plugin.

**Comment by alex-karpovich** [comment]
Please read guide here:
https://github.com/epam/TimeBaseGrafanaPlugin

**Reply by ShivaniSK** [reply]
Hello @alex-karpovich ,
As per the above guide, we need to execute below commands in the server to install plugin
`grafana-cli --pluginUrl https://github.com/epam/TimeBaseGrafanaPlugin/releases/download/1.0.11/epam-timebase-datasource.zip plugins install epam-timebase-datasource
`

However, right now, our server does connect to the above url. So we can manually download the plugin from above url and place the directory in the server.
We want to understand the steps to install the plugin by passing the directory path instead of url.

Regards,
Shivani

**Reply by alex-karpovich** [reply]
please refer to Grafana documentation: https://grafana.com/docs/grafana/latest/administration/plugin-management/#install-plugin-on-local-grafana

**Comment by alex-karpovich** [comment]
we do not support Grafana 10 yet.
